### PR TITLE
[SC-450] June 13 Spell

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

**Comment by 0xShaito** [comment]
All tests run locally ✅ 
The address for WEETH is correct ✅ 
The address and parameters used to deploy the WEETH oracle are correct ✅ 
The tests cover the changes sufficiently ✅ 
All of the parameters for the listing of WEETH match the forum post ✅ 
All the variables in the spell are constant (relevant because of the bytecode validation) ✅

File: src/proposals/20240613/SparkEthereum_20240613.sol [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        return listings;
+    }
+
+    function _postExecute()
+        internal override
+    {
+        ICapAutomator(CAP_AUTOMATOR).setSupplyCapConfig({asset: WEETH, max: 50_000, gap: 5_000, increaseCooldown: 12 hours});
+
+        IERC20(WEETH).approve(address(LISTING_ENGINE.POOL()), 1_000_000);
+        LISTING_ENGINE.POOL().deposit(WEETH, 1_000_000, address(this), 0);
```
Comment (0xShaito) [inline src/proposals/20240613/SparkEthereum_20240613.sol:67]: Can we add a comment explaining why we deposit this amount and what it prevents? Would be nice to keep it for future spells

**Review by 0xShaito — APPROVED** [review]

**Comment by RedLikeRosesss** [comment]
Report link: https://docs.google.com/document/d/1gBdQHTzpTVBzndkpy96WVWd8widH24DXk0gG37zcaVM/edit?usp=sharing

**Review by RedLikeRosesss — APPROVED** [review]

**Review by hexonaut — APPROVED** [review]

**Review by 0xShaito — APPROVED** [review]

**Review by RedLikeRosesss — APPROVED** [review]

**Review by 0xShaito — APPROVED** [review]
LGTM!

- Contract is verified and matches the source code
- Tests are all green locally

**Review by hexonaut — APPROVED** [review]

**Comment by RedLikeRosesss** [comment]
LGTM
- Mainnet code matches source code

**Pull request merged**